### PR TITLE
Feature mag field descriptions

### PR DESCRIPTION
**Pull request by rhosking**

### CODE REVIEW

File: observatory-dags/observatory/dags/database/schema/MagConferenceInstances2020-05-21.json [hunk 1]
```diff
@@ -1,74 +1,92 @@
...
     "name": "EndDate", 
-    "type": "DATETIME"
+    "type": "DATETIME",
+    "description": "The end date of the Conference Instance"
   }, 
   {
     "name": "AbstractRegistrationDate", 
-    "type": "DATETIME"
+    "type": "DATETIME",
+    "description": "The Astract registration date for this Conference Instance"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagConferenceInstances2020-05-21.json:45]:
```suggestion
    "description": "The Abstract registration date for this Conference Instance"
```

File: observatory-dags/observatory/dags/database/schema/MagAffiliations2020-05-21.json [hunk 2]
```diff
@@ -1,54 +1,67 @@
...
     "name": "Rank", 
-    "type": "INTEGER"
+    "type": "INTEGER",
+    "description": "The static rank roughly reflects the log probability of an entity being 'important' - https://docs.microsoft.com/en-us/academic-services/graph/resources-faq#what-is-the-rank-value-on-entities"
   }, 
   {
     "name": "NormalizedName", 
-    "type": "STRING"
+    "type": "STRING",
+    "description": "Normlaised name of the Organisation"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagAffiliations2020-05-21.json:15]:
```suggestion
    "description": "Normalised name of the Organisation"
```

File: observatory-dags/observatory/dags/database/schema/MagAffiliations2020-09-01.json [hunk 3]
```diff
@@ -1,59 +1,72 @@
...
+    "type": "INTEGER",
+    "description": "The static rank roughly reflects the log probability of an entity being 'important' - https://docs.microsoft.com/en-us/academic-services/graph/resources-faq#what-is-the-rank-value-on-entities"
+  }, 
   {
-    "name": "NormalizedName",
-    "type": "STRING"
-  },
+    "name": "NormalizedName", 
+    "type": "STRING",
+    "description": "Normlaised name of the Organisation"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagAffiliations2020-09-01.json:15]:
```suggestion
    "description": "Normalised name of the Organisation"
```

File: observatory-dags/observatory/dags/database/schema/MagAuthors2020-05-21.json [hunk 4]
```diff
@@ -1,38 +1,47 @@
...
     "name": "Rank", 
-    "type": "INTEGER"
+    "type": "INTEGER",
+    "description": "The static rank roughly reflects the log probability of an entity being 'important' - https://docs.microsoft.com/en-us/academic-services/graph/resources-faq#what-is-the-rank-value-on-entities"
   }, 
   {
     "name": "NormalizedName", 
-    "type": "STRING"
+    "type": "STRING",
+    "description": "Normlaised name of the Author"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagAuthors2020-05-21.json:15]:
```suggestion
    "description": "Normalised name of the Author"
```

File: observatory-dags/observatory/dags/database/schema/MagConferenceInstances2020-05-21.json [hunk 5]
```diff
@@ -1,74 +1,92 @@
...
     "name": "ConferenceInstanceId", 
-    "type": "INTEGER"
+    "type": "INTEGER",
+    "description": "PRIMARY KEY: Unique Identifier for a Conference Instance"
   }, 
   {
     "name": "NormalizedName", 
-    "type": "STRING"
+    "type": "STRING",
+    "description": "Normlaised name of the Conference"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagConferenceInstances2020-05-21.json:10]:
```suggestion
    "description": "Normalised name of the Conference"
```

File: observatory-dags/observatory/dags/database/schema/MagConferenceSeries2020-05-21.json [hunk 6]
```diff
@@ -1,34 +1,42 @@
...
     "name": "Rank", 
-    "type": "INTEGER"
+    "type": "INTEGER",
+    "description": "The static rank roughly reflects the log probability of an entity being 'important' - https://docs.microsoft.com/en-us/academic-services/graph/resources-faq#what-is-the-rank-value-on-entities"
   }, 
   {
     "name": "NormalizedName", 
-    "type": "STRING"
+    "type": "STRING",
+    "description": "Normlaised name of the Conference Series"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagConferenceSeries2020-05-21.json:15]:
```suggestion
    "description": "Normalised name of the Conference Series"
```

File: observatory-dags/observatory/dags/database/schema/MagFieldsOfStudy2020-05-21.json [hunk 7]
```diff
@@ -1,42 +1,52 @@
...
     "name": "Rank", 
-    "type": "INTEGER"
+    "type": "INTEGER",
+    "description": "The static rank roughly reflects the log probability of an entity being 'important' - https://docs.microsoft.com/en-us/academic-services/graph/resources-faq#what-is-the-rank-value-on-entities"
   }, 
   {
     "name": "NormalizedName", 
-    "type": "STRING"
+    "type": "STRING",
+    "description": "Normlaised name of the Field of Study"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagFieldsOfStudy2020-05-21.json:15]:
```suggestion
    "description": "Normalised name of the Field of Study"
```

File: observatory-dags/observatory/dags/database/schema/MagJournals2020-05-21.json [hunk 8]
```diff
@@ -1,46 +1,57 @@
...
     "name": "Rank", 
-    "type": "INTEGER"
+    "type": "INTEGER",
+    "description": "The static rank roughly reflects the log probability of an entity being 'important' - https://docs.microsoft.com/en-us/academic-services/graph/resources-faq#what-is-the-rank-value-on-entities"
   }, 
   {
     "name": "NormalizedName", 
-    "type": "STRING"
+    "type": "STRING",
+    "description": "Normlaised name of the Journal"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagJournals2020-05-21.json:15]:
```suggestion
    "description": "Normalised name of the Journal"
```

File: observatory-dags/observatory/dags/database/schema/MagAffiliations2020-05-21.json [hunk 9]
```diff
@@ -1,54 +1,67 @@
...
+  {
+    "name": "CitationCount", 
+    "type": "INTEGER",
+    "description": "The sum of citations for Papers that have been linked to this Organisation"
   },
   {
     "name": "Latitude", 
-    "type": "FLOAT"
+    "type": "FLOAT",
+    "description": "The Latitute of the Organisation"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagAffiliations2020-05-21.json:55]:
```suggestion
    "description": "The Latitude of the Organisation"
```

File: observatory-dags/observatory/dags/database/schema/MagAffiliations2020-09-01.json [hunk 10]
```diff
@@ -1,59 +1,72 @@
...
     "type": "STRING",
     "description": "Two-letter codes (alpha-2) defined in ISO_3166 Code (ISO.org) and ISO 3166-2 (Wiki). Example: US"
   },
   {
-    "name": "Latitude",
-    "type": "FLOAT"
-  },
+    "name": "Latitude", 
+    "type": "FLOAT",
+    "description": "The Latitute of the Organisation"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagAffiliations2020-09-01.json:60]:
```suggestion
    "description": "The Latitude of the Organisation"
```

File: observatory-dags/observatory/dags/database/schema/MagConferenceInstances2020-05-21.json [hunk 11]
```diff
@@ -1,74 +1,92 @@
...
     "name": "CitationCount", 
-    "type": "INTEGER"
+    "type": "INTEGER",
+    "description": "The sum of citations for Papers that have been linked to this Conference Instance"
   }, 
   {
     "name": "Latitude", 
-    "type": "FLOAT"
+    "type": "FLOAT",
+    "description": "The Latitute of the Conference Instance"
```
Comment (aroelo) [inline observatory-dags/observatory/dags/database/schema/MagConferenceInstances2020-05-21.json:80]:
```suggestion
    "description": "The Latitude of the Conference Instance"
```

**Review by aroelo — COMMENTED** [review]
Just some typo fixes :-)

**Comment by rhosking** [comment]
thanks @aroelo! I accepted all the typos

**Review by jdddog — APPROVED** [review]
Nice work Richard!

**Pull request merged**